### PR TITLE
Update Proc stats search in Kibana 

### PR DESCRIPTION
**Pull request by monicasarbu**
As the type changed from `type: proc` to `type: process` in Topbeat, we need to change the  Proc stats search in Kibana. Additionally I deleted two widgets from the Topbeat dashboard as they are not too useful.

### CODE REVIEW

File: dashboards/dashboard/Topbeat-Dashboard.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
+  "panelsJSON": "[{\"col\":1,\"id\":\"Navigation\",\"row\":1,\"size_x\":3,\"size_y\":4,\"type\":\"visualization\"},{\"col\":4,\"id\":\"System-load\",\"row\":1,\"size_x\":5,\"size_y\":4,\"type\":\"visualization\"},{\"col\":9,\"id\":\"Disk-usage-overview\",\"row\":1,\"size_x\":4,\"size_y\":4,\"type\":\"visualization\"},{\"col\":8,\"id\":\"Process-status\",\"row\":5,\"size_x\":5,\"size_y\":4,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Memory-usage\",\"row\":9,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Disk-usage\",\"row\":13,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":7,\"id\":\"CPU-usage\",\"row\":9,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":7,\"id\":\"CPU-usage-per-process\",\"row\":17,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Memory-usage-per-process\",\"row\":13,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Top-processes\",\"row\":17,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Servers\",\"row\":5,\"size_x\":7,\"size_y\":4,\"type\":\"visualization\"}]",
```
Comment (ruflin) [inline dashboards/dashboard/Topbeat-Dashboard.json:7]: Are the changes in this file needed?

File: dashboards/search/Proc-stats.json [hunk 2]
```diff
@@ -8,9 +8,9 @@
   "title": "Proc stats", 
   "version": 1, 
   "kibanaSavedObjectMeta": {
-    "searchSourceJSON": "{\"index\":\"[topbeat-]YYYY.MM.DD\",\"query\":{\"query_string\":{\"query\":\"type: proc\",\"analyze_wildcard\":true}},\"highlight\":{\"pre_tags\":[\"@kibana-highlighted-field@\"],\"post_tags\":[\"@/kibana-highlighted-field@\"],\"fields\":{\"*\":{}},\"require_field_match\":false,\"fragment_size\":2147483647},\"filter\":[]}"
+    "searchSourceJSON": "{\"index\":\"[topbeat-]YYYY.MM.DD\",\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}},\"highlight\":{\"pre_tags\":[\"@kibana-highlighted-field@\"],\"post_tags\":[\"@/kibana-highlighted-field@\"],\"fields\":{\"*\":{}},\"fragment_size\":2147483647},\"filter\":[{\"meta\":{\"negate\":false,\"index\":\"[topbeat-]YYYY.MM.DD\",\"key\":\"type\",\"value\":\"process\",\"disabled\":false},\"query\":{\"match\":{\"type\":{\"query\":\"process\",\"type\":\"phrase\"}}}}]}"
```
Comment (ruflin) [inline dashboards/search/Proc-stats.json]: There are quite a few additional changes in here. Is this because of different Kibana versions?

**Comment by ruflin** [comment]
I recommend to only make the required changes before GA for type:process change to have the change as minimal as possible.

File: dashboards/dashboard/Topbeat-Dashboard.json (hunk 1 above)
Comment (monicasarbu) [inline dashboards/dashboard/Topbeat-Dashboard.json:7]: yes, I removed two widgets that are not useful. I only deleted them from the dashboard for now.

File: dashboards/search/Proc-stats.json (hunk 2 above)
Comment (monicasarbu) [inline dashboards/search/Proc-stats.json]: It is different now as it depends on the way I created the search filter. This time,I saved it from the filters and not from the search. I am fixing it to be the same as previous. Thanks!

**Comment by tsg** [comment]
LGTM, I can live with the extra changes, but please also test with Kibana 4.3 snapshot.

**Pull request merged**